### DISCUSSION TITLE
DLL load failed while importing _socket

### DISCUSSION THREAD

**Discussion by zhaoxian1989** [microsoft/vscode-jupyter#11929] [Questions and Answers]
I want to use the python interactive in vscode, however I found there is a folder, if I add it to the workspace at firstly, vscode will work correct. if I delete this folder and restart vscode, when I start the interactive python by ctrl+shift+P, an error will happen"DLL load failed while importing _socket".
I am not sure what to do if I just want to use interactive python without the folder deleted from the workspace.

**Comment by DonJayamanne** [comment]
> f I add it to the workspace at firstly, vscode will work correct. if I delete this folder and restart vscode, when I start the interactive python by ctrl+shift+P, an error will happen"DLL load failed while importing _socket".

Sounds like you are deleting some folders used by the Python runtime itself.
Pleaes could you 
* share a screenshot of the contents of the folder
* provide details of the name of the folder
* Provide output from the `Jupyter` output panel
